sale_promotion_discount_in_field/wizard/sale_coupon_apply_code.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models
+from odoo.osv import expression
+
+
+class SaleCouponApplyCode(models.TransientModel):
+    _inherit = "sale.coupon.apply.code"
+
+    def apply_coupon(self, order, coupon_code):
+        # OVERRIDE to apply the program to the order without reward lines
+        error_status = {}
```

Comment:
I think this variable not need set because error_status use, if program is exists.

Reply:
resolved